chemicaltoolbox/autodock_vina/.shed.yml

```diff
@@ -0,0 +1,9 @@
+categories:
```

Comment:
This file needs to be either in every sub-folder or you can create suites like this one: https://github.com/galaxyproject/tools-iuc/tree/master/tools/datamash

A suite is more or less a combination of related tools that share a macro file and a common test-data folder. planemo understands by a slightly more complex .shed.yml (https://github.com/galaxyproject/tools-iuc/blob/master/tools/datamash/.shed.yml) how to create for every single tool one extra repo.
A suite has the big advantage of sharing test-data. I think you also have some pdbqt file that you can share between tools, isn't it.

It's up to you :)
If you decide to go for a suite, copy everything into one folder and enhance this one shed.yml file to the extended suite format.